riscv/riscv.ac

```diff
@@ -24,7 +24,7 @@ AC_ARG_WITH(varch,
 	[AS_HELP_STRING([--with-varch=vlen:128,elen:64],
 		[Sets the default vector config])],
   AC_DEFINE_UNQUOTED([DEFAULT_VARCH], "$withval", [Default value for --varch switch]),
-  AC_DEFINE_UNQUOTED([DEFAULT_VARCH], ["vlen:128,elen:64"], [Default value for --varch switch]))
+  AC_DEFINE([DEFAULT_VARCH], ["vlen:128,elen:64"], [Default value for --varch switch]))
```

Comment:
The commit message needs to explain what was broken here, and how this fixes it.

Reply:
The commit message has been updated